### PR TITLE
Add io.github.koromelodev.mindmate

### PR DESCRIPTION
**Pull request by KoromeloDev**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by KoromeloDev** [comment]
bot, build io.github.koromelodev.mindmate

File: io.github.koromelodev.mindmate.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+modules:
+  - name: MindMate
+    buildsystem: cmake
```
Comment (hfiguiere) [inline io.github.koromelodev.mindmate.yml:16]: cmake-ninja?

**Comment by KoromeloDev** [comment]
bot, build io.github.koromelodev.mindmate

File: io.github.koromelodev.mindmate.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+sdk: org.kde.Sdk
+command: MindMate
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-config
```
Comment (hfiguiere) [inline io.github.koromelodev.mindmate.yml:14]: this is not allowed.

Comment (KoromeloDev) [reply io.github.koromelodev.mindmate.yml:14]: How can I access the config folder?

Comment (hfiguiere) [reply io.github.koromelodev.mindmate.yml:14]: application have access to their config directory which is physically in `~/.var/app/$FLATPAK_ID` and proper XDG directories support will find it.

so what are you trying to do?

**Comment by KoromeloDev** [comment]
bot, build io.github.koromelodev.mindmate

**Comment by KoromeloDev** [comment]
bot, build io.github.koromelodev.mindmate

**Comment by KoromeloDev** [comment]
Do I need to add the line `Terminal=true` to the desktop file for the UI application?

**Comment by KoromeloDev** [comment]
bot, build io.github.koromelodev.mindmate

**Comment by KoromeloDev** [comment]
Successful build

**Comment by hfiguiere** [comment]
bot, build io.github.koromelodev.mindmate

**Pull request closed without merging**